config/rbac/auth_proxy_service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8443"
+    prometheus.io/scheme: https
+    prometheus.io/scrape: "true"
+  labels:
+    control-plane: capa-controller-manager
+  name: controller-manager-metrics-service
```

Comment:
Rechecking, this should not be part of the manifests we release, it is optional for metrics. 

Let's remove this whole file. My bad.

Reply:
Removed!